presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -2142,6 +2203,30 @@ else if (column.isHidden()) {
         return columnHandles.build();
     }
 
+    private static void validateCsvColumns(ConnectorTableMetadata tableMetadata)
+    {
+        if (getHiveStorageFormat(tableMetadata.getProperties()) != HiveStorageFormat.CSV) {
+            return;
+        }
+
+        List<String> partitionedBy = getPartitionedBy(tableMetadata.getProperties());
+        List<ColumnMetadata> notSupportedCsvColumns = tableMetadata.getColumns().stream()
```

Comment:
Let's call this `unsupportedColumns` to be consistent with `joinedUnsupportedColumns`. Alternatively, just rename the latter to `joined` since it's of limited scope